CDCgov/prime-reportstream
# 1913 Add sender ID and test kit info to metadata table

This PR adds sender ID and test kit info to metadata table.

Test steps:
1. Update your database by running `./gradlew package`.  This will update the DB and regenerate the Java database files.
2. Run ReportStream
3. Run the smoke tests to generate data - `./gradlew testsmoke`
4. Inspect at least one row of data in the covid_result_metadata table has a sender_id, test_kit_name_id and/or test_performed_loinc_code.  Note that not all data has all this fields, hence it will be empty for some of it.

## Changes
- Added the sender_id, test_kit_name_id and test_performed_loinc_code to the covid_result_metadata table
- Minor tweaks and cleanup of the gradle build file

## Checklist

### Testing

### Process

## Fixes
- #1913

## To Be Done
